components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/Resource.common.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020-2022 JetBrains s.r.o. and respective authors and developers.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package org.jetbrains.compose.resources
+
+/**
+ * Should implement equals() and hashCode()
```

Comment:
As it is public API, I believe we should write more about this abstract interface.

Reply:
Sure, we can do it later